<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1277</width>
    <height>905</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>281</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTreeWidget" name="structTree">
         <column>
          <property name="text">
           <string>控制结构</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="pageTree">
         <column>
          <property name="text">
           <string>网页动作</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="elementTree">
         <column>
          <property name="text">
           <string>元素</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="middle" native="true"/>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="maximumSize">
       <size>
        <width>281</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTreeWidget" name="rightTree">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>模板</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="selectDemoButton">
            <property name="text">
             <string>选择模板</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveDemoButton">
            <property name="text">
             <string>保存模板</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="taskTree">
         <column>
          <property name="text">
           <string>执行</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="selectTaskButton">
            <property name="text">
             <string>选择模板</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pauseTaskButton">
            <property name="text">
             <string>暂停</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="startTaskButton">
            <property name="text">
             <string>继续/执行</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1277</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="elementMenu">
    <property name="title">
     <string>元素动作</string>
    </property>
   </widget>
   <widget class="QMenu" name="chromeMenu">
    <property name="title">
     <string>浏览器选项</string>
    </property>
   </widget>
   <widget class="QMenu" name="pageMenu">
    <property name="title">
     <string>网页动作</string>
    </property>
   </widget>
   <widget class="QMenu" name="structMenu">
    <property name="title">
     <string>控制结构</string>
    </property>
   </widget>
   <addaction name="structMenu"/>
   <addaction name="elementMenu"/>
   <addaction name="pageMenu"/>
   <addaction name="chromeMenu"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBind"/>
   <addaction name="actionRemoveElements"/>
   <addaction name="actionHelp"/>
  </widget>
  <action name="actionBind">
   <property name="text">
    <string>绑定</string>
   </property>
   <property name="toolTip">
    <string>Tags bind</string>
   </property>
  </action>
  <action name="actionRemoveElements">
   <property name="text">
    <string>清空元素</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <signal>signal1()</signal>
  <slot>middleClick()</slot>
 </slots>
</ui>
